index.js

```diff
@@ -1,4 +1,2 @@
 'use strict';
-const cryptoRandomString = require('crypto-random-string');
-
-module.exports = () => cryptoRandomString(32);
+module.exports = () => require('crypto').randomBytes(32).toString('hex');
```

Comment:
This changes the behavior. At least follow the existing implementation: https://github.com/sindresorhus/crypto-random-string/blob/9b3a4704ab94a94f48197c4eb9c7409829b279a3/index.js#L66